### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by alisonpchase**
admonition removed; brief content added to describe and link to each section within Resources.

### CODE REVIEW

**Comment by alisonpchase** [comment]
It appears my links to internal pages didn't work (e.g., I tried to link the 'Preparation' page) - @emiliom do you mind taking a look when you get a chance to see what I did wrong? thanks!

File: resources/index.md [hunk 1]
```diff
@@ -1,20 +1,26 @@
...
 **This page should be rewritten entirely!**
-```
+```-->
 
-To make sure that OHW22 will be a welcoming environment for everyone, please read our [Code of Conduct](/about/code-of-conduct.md) carefully as part of your preparation. We expect all participants to adhere to the Code of Conduct in all interactions throughout the hackweek.
+To make sure that OHW23 will be a welcoming environment for everyone, please read our [Code of Conduct](/about/code-of-conduct.md) carefully as part of your preparation. We expect all participants to adhere to the Code of Conduct in all interactions throughout the hackweek.
 
 ## Preparation
 
+A variety of content can be found under the [Preparation](/prep/index.md) page, with resources provided on Git, GitHub, JupyterHub, Conda, Python and R.
```
Comment (emiliom) [inline resources/index.md:11]: Remove the starting `/`. The url path is relative to the current page.
```suggestion
A variety of content can be found under the [Preparation](prep/index.md) page, with resources provided on Git, GitHub, JupyterHub, Conda, Python and R.
```

File: resources/index.md [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+We will use Slack, Zoom and GitHub as our primary communication and content delivery platforms. The week before OceanHackWeek you will receive via email an invitation to join the OceanHackWeek GitHub organization, [https://github.com/oceanhackweek/](https://github.com/oceanhackweek/) if you have not received it already, along with an invitation to join our Slack organization.
+
+## Projects
+
+The [Projects](/projects.md) page provides some tips and definitions regarding the motivation behind a "hack" event, what a hack project is, and how to brainstorm a project that would be a good fit for the hackweek setting.
 
 ## Organizing OceanHackWeek 
 
-PLACEHOLDER. SEE THE Organizing OHW section
+The [Organizing OHW](/organizing/index.md) page provides resources for project mentors and tutorial helpers who are involved in the facilitation of the hackweek event. If either of these roles apply to you, please read through the information provided, and thank you!
```
Comment (emiliom) [inline resources/index.md:23]: Remove the starting `/`. The url path is relative to the current page.
```suggestion
The [Organizing OHW](organizing/index.md) page provides resources for project mentors and tutorial helpers who are involved in the facilitation of the hackweek event. If either of these roles apply to you, please read through the information provided, and thank you!
```

**Comment by emiliom** [comment]
Thanks! I've suggested two changes, both of the same type. I'll go ahead and commit them so the new preview can auto-generate and we confirm whether they fixed the issue.

File: resources/index.md [hunk 3]
```diff
@@ -1,20 +1,26 @@
...
 
-We will use Slack, Zoom and GitHub as our primary communication and content delivery platforms. The week before OceanHackWeek you will receive via email an invitation to join the OceanHackWeek GitHub organization, [https://github.com/oceanhackweek/](https://github.com/oceanhackweek/) if you have not received it already, along with Slack invites and Zoom connection info.
+We will use Slack, Zoom and GitHub as our primary communication and content delivery platforms. The week before OceanHackWeek you will receive via email an invitation to join the OceanHackWeek GitHub organization, [https://github.com/oceanhackweek/](https://github.com/oceanhackweek/) if you have not received it already, along with an invitation to join our Slack organization.
+
+## Projects
+
+The [Projects](/projects.md) page provides some tips and definitions regarding the motivation behind a "hack" event, what a hack project is, and how to brainstorm a project that would be a good fit for the hackweek setting.
```
Comment (emiliom) [inline resources/index.md:19]: I forgot to change this one, too:
```suggestion
The [Projects](projects.md) page provides some tips and definitions regarding the motivation behind a "hack" event, what a hack project is, and how to brainstorm a project that would be a good fit for the hackweek setting.
```

**Comment by alisonpchase** [comment]
Super, thanks! It looks good to me so I will merge it in.

**Pull request merged**